### PR TITLE
Extracted drawing logic into separate component

### PR DESCRIPTION
**Pull request by kateinkim**
#### 1. Changes
- The types and interfaces are moved to `types.ts` file from `AILabImage.tsx`
- The logic for drawing boundary boxes is also moved into its own component

#### 2. Photo/Video
CleanShot 2021-12-28 at 14 36 08

### CODE REVIEW

File: packages/ai-lab/src/components/AILabImage/AILabImage.tsx [hunk 1]
```diff
@@ -1,129 +1,31 @@
...
-import { CLASSES } from '../labels';
-import {
-  Performance,
-  PerformanceInfo,
-  perfInfo,
-  PerformanceProps,
-} from '../../performance';
+import { Performance, PerformanceInfo, perfInfo } from '../../performance';
+import { ImageProps, ModelInfo } from '../../types';
+import { DrawDetections } from '../DrawDetections';
```
Comment (GantMan) [inline packages/ai-lab/src/components/AILabImage/AILabImage.tsx:5]: Let's rename to AILabObjectDetctionUI

**Review by GantMan — CHANGES_REQUESTED** [review]
Allow `customDetectionUI` to be a prop of configuration.

Also, let's rename into AILab style

File: packages/ai-lab/src/components/AILabImage/AILabImage.tsx (hunk 1 above)
Comment (kateinkim) [reply packages/ai-lab/src/components/AILabImage/AILabImage.tsx]: Fixed!

**Comment by kateinkim** [comment]
I hope this is how it's supposed to be 😃

CleanShot 2021-12-29 at 14 55 41

**Comment by GantMan** [comment]
giphy

**Pull request merged**